### PR TITLE
Remove obsolete members from v5

### PR DESCRIPTION
**Pull request by Shane32**

### CODE REVIEW

**Comment by sungam3r** [comment]
I will postpone the review of this PR to first familiarize myself with earlier changes.

File: docs2/site/docs/migrations/migration5.md [hunk 1]
```diff
@@ -47,3 +47,26 @@ property containing the execution variables has now been renamed to `Variables`.
...
 
 Also, `IConfigureExecution` renamed to `IConfigureExecutionOptions`.
+
+### Classes and members marked as obsolete have been removed
+
+The following classes and members that were marked with `[Obsolete]` in v4 have been removed:
+
+| Class or member | Notes |
+|-----------------|-------|
+| GraphQL.NewtonsoftJson.StringExtensions.GetValue     |                             |
```
Comment (sungam3r) [inline docs2/site/docs/migrations/migration5.md:57]: "`" for all names

File: src/GraphQL.NewtonsoftJson/StringExtensions.cs [hunk 2]
```diff
@@ -87,15 +73,6 @@ public static T FromJson<T>(this System.IO.Stream stream)
...
-        /// <summary>
-        /// Gets the value contained in a JObject, JValue, JProperty or JArray.
-        /// </summary>
-        /// <param name="value">The object containing the value to extract.</param>
-        /// <remarks>If the value is a recognized type, it is returned unaltered.</remarks>
-        [Obsolete("This method will be removed in a future version of GraphQL.NET.")]
-        public static object GetValue(this object value)
-            => GetValueInternal(value);
-
         private static object GetValueInternal(this object value)
```
Comment (sungam3r) [inline src/GraphQL.NewtonsoftJson/StringExtensions.cs:76]:
```suggestion
        private static object GetValue(this object value)
```

File: src/GraphQL.SystemTextJson/ObjectDictionaryConverter.cs [hunk 3]
```diff
@@ -1,24 +0,0 @@
...
-    public class ObjectDictionaryConverter : JsonConverter<Dictionary<string, object>>
-    {
-        /// <inheritdoc/>
-        public override void Write(Utf8JsonWriter writer, Dictionary<string, object> value, JsonSerializerOptions options)
-            => throw new NotImplementedException(
-                "This converter currently is only intended to be used to read a JSON object into a strongly-typed representation.");
-
-        /// <inheritdoc/>
-        public override Dictionary<string, object> Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
-            => InputsConverter.ReadDictionary(ref reader);
```
Comment (sungam3r) [inline src/GraphQL.SystemTextJson/ObjectDictionaryConverter.cs:22]: Mark `InputsConverter.ReadDictionary` private.

File: src/GraphQL.Tests/Serialization/SystemTextJson/ObjectDictionaryConverterTests.cs [hunk 4]
```diff
@@ -1,297 +0,0 @@
-using System.Collections.Generic;
-using System.Text.Encodings.Web;
-using System.Text.Json;
-using GraphQL.SystemTextJson;
-using Shouldly;
-using Xunit;
-
-namespace GraphQL.Tests.Serialization.SystemTextJson
-{
-    public class ObjectDictionaryConverterTests
```
Comment (sungam3r) [inline src/GraphQL.Tests/Serialization/SystemTextJson/ObjectDictionaryConverterTests.cs:10]: Do we have tests for `InputsConverter`? InputsConverterTests, right?

File: src/GraphQL.Tests/Shared/JsonStringExtensions.cs [hunk 5]
```diff
@@ -19,5 +20,17 @@ public static ExecutionResult ToExecutionResult(this string json, ExecutionError
                 Errors = errors,
                 Executed = executed
             };
+
+        public static Dictionary<string, object> ToDictionary(this string json)
```
Comment (sungam3r) [inline src/GraphQL.Tests/Shared/JsonStringExtensions.cs:24]: Nullable annotations.

Comment (sungam3r) [reply src/GraphQL.Tests/Shared/JsonStringExtensions.cs:24]: Or just delete method and use `object converted = SystemTextJson.StringExtensions.ToInputs($"{{ \"arg\": {valueString} }}")["arg"];`in AllScalarGraphTypeTests.cs 

?

File: src/GraphQL.Tests/Serialization/SystemTextJson/ObjectDictionaryConverterTests.cs (hunk 4 above)
Comment (Shane32) [reply src/GraphQL.Tests/Serialization/SystemTextJson/ObjectDictionaryConverterTests.cs:10]: Yes

File: src/GraphQL.Tests/Shared/JsonStringExtensions.cs (hunk 5 above)
Comment (Shane32) [reply src/GraphQL.Tests/Shared/JsonStringExtensions.cs]: Added NRT anntotations

File: src/GraphQL.SystemTextJson/ObjectDictionaryConverter.cs (hunk 3 above)
Comment (Shane32) [reply src/GraphQL.SystemTextJson/ObjectDictionaryConverter.cs:22]: Done

File: src/GraphQL.NewtonsoftJson/StringExtensions.cs (hunk 2 above)
Comment (Shane32) [reply src/GraphQL.NewtonsoftJson/StringExtensions.cs]: Done

File: docs2/site/docs/migrations/migration5.md (hunk 1 above)
Comment (Shane32) [reply docs2/site/docs/migrations/migration5.md]: Done

**Review by sungam3r — APPROVED** [review]
👍 I am glad that we finally got rid of all of that code.

**Pull request merged**